doc/python/webgl-vs-svg.md

```diff
@@ -58,6 +59,15 @@ In addition to the above limitations, the WebGL-powered version of certain SVG-p
 * Range breaks on time-series axes are not yet supported
 * Axis range heuristics may differ
 
+### Multiple WebGL Contexts
+
+*New in 5.19*
+
+If you encounter WebGL context limits when rendering many WebGL-based figures on one page, you can use [Virtual WebGL](https://github.com/greggman/virtual-webgl).
+
+To use it, in the environment where your Plotly figures are being rendered, load the Virtual WebGL script, "https://unpkg.com/virtual-webgl@1.0.6/src/virtual-webgl.js", for example, using a `<script>` tag.
+
```

Comment:
Would you provide an example of using the script?

Reply:
Most importantly, how do you include this script in Jupyter (lab or classic notebook)? Is there a docs page from Jupyter that covers this?

Reply:
I've added how to do it by running a cell as a HTML cell with a magic command and linked out to the docs. When testing, this was able to load the script in